lib/features/chat/domain/helix_get_channel_chat_badges.dart

```diff
@@ -0,0 +1,39 @@
+import 'package:freezed_annotation/freezed_annotation.dart';
+
+part 'helix_get_channel_chat_badges.freezed.dart';
+part 'helix_get_channel_chat_badges.g.dart';
+
+@freezed
+class HelixGetChannelChatBadgesResponse
```

Comment:
The endpoint for channel and global chat badges share the same response, this can be converted into a single `HelixGetChatBadgeResponse` data class to prevent duplication.